ppx/html.ml

```diff
@@ -15,558 +17,1859 @@ type eventType =
-  ; Attribute {type_= String; name= "vocab"; htmlName= "vocab"}
-  ; Attribute {type_= String; name= "width"; htmlName= "width"}
-  ; Attribute {type_= String; name= "widths"; htmlName= "widths"}
-  ; Attribute {type_= String; name= "wordSpacing"; htmlName= "wordSpacing"}
-  ; Attribute {type_= String; name= "wrap"; htmlName= "wrap"}
-  ; Attribute {type_= String; name= "writingMode"; htmlName= "writingMode"}
-  ; Attribute {type_= String; name= "x"; htmlName= "x"}
-  ; Attribute {type_= String; name= "x1"; htmlName= "x1"}
-  ; Attribute {type_= String; name= "x2"; htmlName= "x2"}
-  ; Attribute
-      {type_= String; name= "xChannelSelector"; htmlName= "xChannelSelector"}
-  ; Attribute {type_= String; name= "xHeight"; htmlName= "xHeight"}
-  ; Attribute {type_= String; name= "xlinkActuate"; htmlName= "xlinkActuate"}
-  ; Attribute {type_= String; name= "xlinkArcrole"; htmlName= "xlinkArcrole"}
-  ; Attribute {type_= String; name= "xlinkHref"; htmlName= "xlinkHref"}
-  ; Attribute {type_= String; name= "xlinkRole"; htmlName= "xlinkRole"}
-  ; Attribute {type_= String; name= "xlinkShow"; htmlName= "xlinkShow"}
-  ; Attribute {type_= String; name= "xlinkTitle"; htmlName= "xlinkTitle"}
-  ; Attribute {type_= String; name= "xlinkType"; htmlName= "xlinkType"}
-  ; Attribute {type_= String; name= "xmlBase"; htmlName= "xmlBase"}
-  ; Attribute {type_= String; name= "xmlLang"; htmlName= "xmlLang"}
-  ; Attribute {type_= String; name= "xmlns"; htmlName= "xmlns"}
-  ; Attribute {type_= String; name= "xmlnsXlink"; htmlName= "xmlnsXlink"}
-  ; Attribute {type_= String; name= "xmlSpace"; htmlName= "xmlSpace"}
-  ; Attribute {type_= String; name= "y"; htmlName= "y"}
-  ; Attribute {type_= String; name= "y1"; htmlName= "y1"}
-  ; Attribute {type_= String; name= "y2"; htmlName= "y2"}
-  ; Attribute
-      {type_= String; name= "yChannelSelector"; htmlName= "yChannelSelector"}
-  ; Attribute {type_= String; name= "z"; htmlName= "z"}
-  ; Attribute {type_= String; name= "zoomAndPan"; htmlName= "zoomAndPan"}
-  ; Event {name= "onAbort"; type_= Media}
-  ; Event {name= "onAnimationEnd"; type_= Animation}
-  ; Event {name= "onAnimationIteration"; type_= Animation}
-  ; Event {name= "onAnimationStart"; type_= Animation}
-  ; Event {name= "onBlur"; type_= Focus}
-  ; Event {name= "onCanPlay"; type_= Media}
-  ; Event {name= "onCanPlayThrough"; type_= Media}
-  ; Event {name= "onChange"; type_= Form}
-  ; Event {name= "onClick"; type_= Mouse}
+type element = {tag: string; attributes: prop list}
+
+let attributeReferrerPolicy = String
+(* | Empty
+   | NoReferrer
+   | NoReferrerWhenDowngrade
+   | Origin
+   | OriginWhenCrossOrigin
+   | SameOrigin
+   | StrictOrigin
+   | StrictOriginWhenCrossOrigin
+   | UnsafeUrl *)
+
+let attributeAnchorTarget = String
+(* | Self
+   | Blank
+   | Parent
+   | Top
+   | Custom of String *)
+
+let domAttributes =
+  [ Event {name= "onCopy"; type_= Clipboard}
```

Comment:
I wonder about OCaml syntax future support. Do you think we should use some live conversion from `name`, replacing capital letters with underscore? e.g. from `onCopy` to `on_copy`. One alternative could be to add another field `ocaml_name` to the record, but it seems to add a lot of boilerplate 😬

Reply:
If we want to provide both names, I would say to store it in one form and transform it during the ppx injection. WDYT?